examples/api-resource/main.go

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/asgardeo/go/pkg/config"
+	"github.com/asgardeo/go/pkg/sdk"
+)
+
+func main() {
+
+	// Initialize the client configurations.
+	cfg := config.DefaultClientConfig().
+		WithBaseURL("https://api.asgardeo.io/t/<tenant-domain>").
+		WithTimeout(10*time.Second).
+		WithClientCredentials(
+			"client_id",
+			"client_secret",
+		)
+
+	// Create a client with the given configurations.
```

Comment:
Do we need these comment. IMO code is self explanatory. So let's remove redundant comments and add where necessary ?